### PR TITLE
Milestone cleanup

### PR DESCRIPTION
**Pull request by MakMuftic**
I fixed explorers url-s as now westend is not supported on polkascan.

Everything else seems to be working, but I am still investigating #79 as it messes with snap after few operations (` MaxListenersExceededWarning`). 

If you look _relevant code from Polkadot API_ that I mentioned inside issue #79 you will see that `promiseTracker` function is used and that unsubscribe callback fails with the illegal invocation error. This is interesting as this same mechanism (creating tracker with `promiseTracker` function) is used when subscribing to different events inside Polkadot API, and it resolves with the same error (because of this we made the last fix on snaps-cli). 

In conclusion I think we will fix more than one problem by figuring out why this happens. Unfortunately, [issue](https://github.com/MetaMask/metamask-snaps-beta/issues/133) on Metamask repo to return more meaningful error message is still not resolved but I am investigating this.

### CODE REVIEW

File: packages/snap/src/asset/index.ts [hunk 1]
```diff
@@ -8,13 +8,29 @@ import {SnapConfig} from "@nodefactory/metamask-polkadot-types";
 
 export const POLKADOT_SNAP_ASSET_IDENTIFIER = "polkadot-snap-asset";
 
+function getCustomViewUrl(configuration: SnapConfig, address: string): string {
```
Comment (mpetrunic) [inline packages/snap/src/asset/index.ts:11]: do we even need that since tx are in state?

Comment (MakMuftic) [reply packages/snap/src/asset/index.ts:11]: @mpetrunic you are right, especially because it is possible to set this trough configuration.

Comment (mpetrunic) [reply packages/snap/src/asset/index.ts:11]: i dont think we even need explorer url in snap

Comment (MakMuftic) [reply packages/snap/src/asset/index.ts:11]: I am not sure where in the snap are you referring? Because after refactoring this, we won't be using explorer anywhere inside snap. 

The only usage is inside _example_ (to provide a link on the executed transaction on explorer)

**Comment by MakMuftic** [comment]
@mpetrunic I wasn't able to resolve the mentioned issue. I will write today an extended explanation of the problem to the Polkadot team, so maybe they can help us resolve this recurring issue, but you can maybe release this as it is.
